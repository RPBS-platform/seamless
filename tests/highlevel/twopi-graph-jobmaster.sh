rm -f twopi-colored-jobmaster.seamless

python3 simple-pi-database.py
python3 ~/seamless-scripts/jobslave.py --database --time 5 &
sleep 2
python3 ~/seamless-scripts/color-graph-jobmaster.py twopi.seamless twopi-colored-jobmaster.seamless
wait
md5sum twopi-colored-jobmaster.seamless
