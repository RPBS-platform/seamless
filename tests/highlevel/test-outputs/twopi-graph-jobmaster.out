d89a78eede0f535003eae21099499943  twopi.seamless
d89a78eede0f535003eae21099499943  /tmp/twopi-database.seamless
OUTGOING jobslave jobmaster
INCOMING jobmaster jobslave
ADD SERVANT buffer
ADD SERVANT buffer
ADD SERVANT buffer_length
ADD SERVANT buffer_length
ADD SERVANT semantic_to_syntactic
ADD SERVANT semantic_to_syntactic
ADD SERVANT transformation
Set up a communion outgoing port 8602
REMOVE SERVANT buffer
REMOVE SERVANT buffer_length
REMOVE SERVANT semantic_to_syntactic
956e1f34a0047ad5f53dea731af782bd  twopi-colored-jobmaster.seamless
